client/src/components/pages/HomeWork/data.js

```diff
@@ -0,0 +1,15 @@
+const data = [
+  {
+    id: 1,
+    Homework_description:
```

Comment:
I know this is just fake data, but pls use camelCase for object properties (same for HmURLS)

Reply:
Done